node/src/options.rs

```diff
@@ -86,6 +80,50 @@ pub struct RunOptions {
     pub rpc_addr: Option<SocketAddr>,
 }
 
+/// Run the regtest node.
+#[derive(Args, Debug)]
```

Comment:
Perhaps this can be moved to its own rs file. No need to mix it with the others here.